dashboard/fixtures/certificates.json

```diff
@@ -5,14 +5,14 @@
 	    "certificate_type": "verified",
 	    "status": "downloadable",
 	    "download_url": "http://www.example.com/demo.pdf",
-	    "grade": "0.98"
+	    "grade": 98
```

Comment:
that fix looks like it's part of an unreleased version. here's their releases page: https://github.com/elastic/elasticsearch/releases. i'm not quite sure what to make of the v2.x.x versions mixed in with the v.5.0.0-x versions (elasticsearch version vs. lucene version??), but it looks like they haven't released that [histogram fix](https://github.com/elastic/elasticsearch/pull/19551) yet (it's tagged as 'v5.0.0-alpha5'). 

i can create an issue to try that version when it releases, make sure it works with decimal values for grades, and revert our code as necessary. would that address your concerns @noisecapella ?